docs/install/index.md

```diff
@@ -124,20 +124,38 @@ Indicate your preferred configuration. Then, follow the customized commands to i
 $ pip install mxnet
 ```
 
+MXNet offers pip packages with MKL-DNN enabled which will be much faster when running on Intel hardware. Try the following command line to install it and find performance numbers and tuning guide in <a href="https://mxnet.incubator.apache.org/versions/master/faq/perf.html#intel-cpu">performance on Intel CPU</a>.
```

Comment:
```suggestion
MKL-DNN enabled pip packages are optimized for Intel hardware. You can find performance numbers in the <a href="../..//faq/perf.md#intel-cpu">MXNet tuning guide</a>.
```

Reply:
Hi @aaronmarkham, the link here will be compiled to https://mxnet.incubator.apache.org/versions/faq/perf.md#intel-cpu which is a blank page. Do you think I can use http://mxnet.io/faq/perf.html#intel-cpu instead here?

Reply:
@pengzhao-intel